src/assets/nonstop.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+<g>
+	<g>
+		<path d="M256,0C114.848,0,0,114.848,0,256s114.848,256,256,256s256-114.848,256-256S397.152,0,256,0z M256,480
+			C132.48,480,32,379.52,32,256S132.48,32,256,32s224,100.48,224,224S379.52,480,256,480z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M256,112c-69.744,0-128.032,49.856-141.184,115.776L92.8,198.432l-25.6,19.2l48,64c3.136,4.192,7.952,6.4,12.816,6.4
+			c3.328,0,6.704-1.04,9.584-3.2l64-48l-19.2-25.6l-36.96,27.728C153.696,185.28,200.048,144,256,144
+			c39.872,0,77.056,21.488,97.04,56.064l27.696-16.016C355.056,139.616,307.264,112,256,112z"/>
+	</g>
+</g>
+<g>
+	<g>
+		<path d="M396.496,246c-5.504-6.896-15.52-8.016-22.4-2.56l-60.8,48l19.824,25.104l33.248-26.256
+			C357.584,343.36,311.504,384,256,384c-39.856,0-77.04-21.456-97.024-56.016l-27.696,16.032C156.976,388.416,204.752,416,256,416
+			c70.368,0,128.992-50.768,141.456-117.568L422.704,330l24.992-19.984L396.496,246z"/>
+	</g>

```

Comment:
can you break down for me how these svg's work.  would love a deep dive on this.